api/src/pcapi/alembic/versions/20240212T113318_e8da34054129_add_updateDate_to_google_places_info.py

```diff
@@ -0,0 +1,22 @@
+"""
+add updateDate to google_places_info
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "e8da34054129"
+down_revision = "d118574a7888"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column("google_places_info", sa.Column("updateDate", sa.DateTime(), nullable=False))
```

Comment:
Cette colonne n'a pas de valeur par défaut côté PostgreSQL, donc l'ajouter avec une contrainte `NOT NULL` n'est pas possible. Il pourrait se passer ce qui suit : 
1. On joue cette migration.
2. Le code (ancienne version, dont le modèle n'a pas connaissance de l'attribut `updateDate`) tente d'ajouter une ligne. Il ne fournit donc pas de valeur pour `updateDate`. PostgreSQL va renvoyer une erreur, car la colonne est non-nullable.

Reply:
est ce qu'il est possible de rajouter une valeur par defaut côté PostgreSQL?

Reply:
et je ne comprend pas pourquoi une ancienne version du code sera éxécuté?

Reply:
oui mais le code est lancé à la main pour le moment donc il ne sera pas lancé entre les deux